oeps/best-practices/oep-0051-bp-conventional-commits.rst

```diff
@@ -88,6 +88,8 @@ We use these commit types labels:
 
 * **revert**: undo a previous change. The previous commit message, including the type label, is included.
 
+* **squash**: temporary changes that are intended to be squashed before merging.
```

Comment:
I might suggest not having this as part of the list, but as an addendum below the list, so that it's not seen as a "regular permitted value" by someone reading the list quickly.

Reply:
You've hit on the ugly truth of pretending that the linting question is separate, when we in fact are actively linting. I agree that it would be best to clarify expectations. I could add something like:
```
  * **Note**: At this time, "squash" commits are a convention, but will still fail the build. See the open questions section if you believe these should pass the build.
```
In the future, we'd remove this note altogether if we chose to make it pass, or we could update the note to:
```
  * **Note**: Although "squash" commits are a convention that convey meaning, they will still fail builds to ensure these commits are not merged.
```

Reply:
Actually, here is a potential update:
```
* **squash**: temporary changes that are intended to be squashed before merging. This label is a convention that conveys meaning, but it will still fail the build. See the tooling open questions section for the pass/fail debate.
```

Note: I liked using "temp" because it would pass the build, and I'm not sure that people won't continue to do so. I can't promise I won't while this is still under debate.

Reply:
I added a comment about warning instead of failing to the linting thread above

Reply:
(As for using other tags just to make the build “green”, I still say that’s just like adding `pylint disable` instead of actually fixing the warning, or `assert True #FIXME` to a unit test)